src/modules/headset/headset_openxr.c

```diff
@@ -1019,6 +1021,20 @@ static float openxr_getDisplayFrequency(void) {
   return frequency;
 }
 
+static float* openxr_getDisplayFrequencies(uint32_t* count) {
+  if (!state.features.refreshRate || !count) return NULL;
+  XR(xrEnumerateDisplayRefreshRatesFB(state.session, 0, count, NULL));
+  float *frequencies = malloc(*count * sizeof(float));
+  lovrAssert(frequencies, "Out of Memory");
+  XR(xrEnumerateDisplayRefreshRatesFB(state.session, *count, count, frequencies));
+  return frequencies;
+}
+
+static bool openxr_setDisplayFrequency(float frequency) {
+  if (!state.features.refreshRate) return false;
+  XR(xrRequestDisplayRefreshRateFB(state.session, frequency));
```

Comment:
I wonder if this should manually handle `XR_ERROR_DISPLAY_REFRESH_RATE_UNSUPPORTED_FB` and return `false` for that instead of throwing an error.

Reply:
Good idea! I updated the patch.